File: FlashCards.Models/DTOs/ToClient/SubscribedCourseShort.cs
using System;

namespace FlashCards.Models.DTOs.ToClient
{
    public class SubscribedCourseShort
    {
        public int Id { get; set; }

        public DateTime LastActivity { get; set; }

        public decimal OverallProgress { get; set; }

        public int CourseId { get; set; }

        public CourseShort Course { get; set; }
    }
}

File: FlashCards-SPA/src/app/core/_models/subscribedCourseShort.ts
export class SubscribedCourseShort {
  subscriptionId: number;
  courseId: number;
  courseName: string;
  courseAccountCreatedName: string;
  overallProgress: number;
  lastActivity: Date;

  constructor(subscriptionId: number, courseId: number, courseName: string,
              courseAccountCreatedName: string, overallProgress: number,
              lastActivity: Date) {
    this.subscriptionId = subscriptionId;
    this.courseId = courseId;
    this.courseName = courseName;
    this.courseAccountCreatedName = courseAccountCreatedName;
    this.overallProgress = overallProgress;
    this.lastActivity = lastActivity;
  }

}
